include/eosio/ship_protocol.hpp

```diff
@@ -124,19 +126,22 @@ namespace eosio { namespace ship_protocol {
 
    using request = std::variant<get_status_request_v0, get_blocks_request_v0, get_blocks_ack_request_v0>;
 
-   struct get_blocks_result_v0 {
+   struct get_blocks_result_base_v0 {
```

Comment:
You can leave off the `_v0`.  This is common to `get_blocks_result_v0` and `get_blocks_result_v1`.